streams/id.m3u

```diff
@@ -23,22 +25,26 @@ https://liveanevia.mncnow.id/live/eds/HitsMovies/sa_dash_vmx/HitsMovies.mpd
 http://iamchannel.org:1935/tes/1/playlist.m3u8
 #EXTINF:-1 tvg-id="IndonesiaChannel.id" status="online",Indonesia Channel (720p) [Not 24/7]
 http://wowzaprod236-i.akamaihd.net/hls/live/1019903/7dd4cf51/playlist.m3u8
-#EXTINF:-1 tvg-id="Indosiar.id" status="blocked",Indosiar [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/205-indosiar
-#EXTINF:-1 tvg-id="" status="online",iNews (720p)
+#EXTINF:-1 tvg-id="Indosiar.id" status="online",Indosiar (720p)
+http://rr2.dens.tv/s/s04/01.m3u8
+#EXTINF:-1 tvg-id="INews.id" status="online",iNews (720p)
 https://vcdn2.rctiplus.id/live/eds/inews_fta/live_fta/inews_fta.m3u8
-#EXTINF:-1 tvg-id="KompasTV.id" status="error",Kompas TV [Geo-blocked]
-http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.vidio.com/live/874-kompas-tv
+#EXTINF:-1 tvg-id="BeritaSatuEnglish.id" status="online",JakartaGlobe News (540p) [Not 24/7]
```

Comment:
Why is the name not corresponding to the tvg-id ?

Reply:
BetitaSatu English was renamed to JakartaGlobe News. Its intended for compatibility